sdk/python/express_relay/svm/limo_client.py

```diff
@@ -111,82 +107,72 @@ def create_associated_token_account_idempotent(
             data=bytes([1]),  # idempotent version of the instruction
         )
 
-    async def get_ata_and_create_ixn_if_required(
+    def get_ata_and_create_ixn_if_required(
         self,
         owner: Pubkey,
         token_mint_address: Pubkey,
         token_program_id: Pubkey,
         payer: Pubkey,
     ) -> Tuple[Pubkey, Sequence[Instruction]]:
         ata = self.get_ata(owner, token_mint_address, token_program_id)
-        if not await self.account_exists(ata):
-            ix = self.create_associated_token_account_idempotent(
-                payer, owner, token_mint_address, token_program_id
-            )
-            return ata, [ix]
-        return ata, []
+        ix = self.create_associated_token_account_idempotent(
```

Comment:
i think this is fine, just double checking that limo sdk does idempotent for general ATAs too?

Reply:
it does, I cross checked